dmprsim/simulator/draw.py

```diff
@@ -2,372 +2,262 @@
 import shutil
 from pathlib import Path
 
+import numpy as np
+
 try:
     import cairo
 except ImportError:
     import cairocffi as cairo
 
+from .middlewares import RouterForwardedPacketMiddleware, \
+    RouterTransmittedMiddleware
+
 IMAGE_WIDTH = 1920
```

Comment:
Switch to 4k here by default?

Reply:
4K is much slower because saving the single images as png is the bottleneck when generating videos, I added an option for different resolutions but made hd the default